contracts/strategies/CurveStrategy.sol

```diff
@@ -272,15 +312,15 @@ contract CurveStrategy is Ownable, IStrategy {
             }
 
             // passes false not to claim rewards
-            gauge.withdraw(poolWithdrawAmount, false);
+            gauge.withdraw(poolWithdrawAmount);
```

Comment:
just wanted to make it consistent with all pools calling `claim_rewards` is common so hence

Reply:
got it. but then we should probably remove the comment about `passes false not to claim rewards`.
but this means that we're still not claiming the rewards? If such argument is really expected, it's better to explicitly pass `false` rather than rely on default values (better readability). what do you think?